src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -4057,12 +4089,17 @@ namespace SimAirServingZones {
 
 				for ( ZonesCooledNum = 1; ZonesCooledNum <= NumZonesCooled; ++ZonesCooledNum ) {
 					CtrlZoneNum = AirToZoneNodeInfo( AirLoopNum ).CoolCtrlZoneNums( ZonesCooledNum );
+					int TermUnitSizingIndex = AirToZoneNodeInfo( AirLoopNum ).TermUnitCoolSizingIndex( ZonesCooledNum );
```

Comment:
<sigh> Yes, it's a bit twisted, but it's the only way I could get it to work with the scattered getinputs.  A refactor of all that would simplify this, but what I have now is the ZoneEquipConfig.AirDistUnitCool (and Heat).TermUnitSizingIndex are all preassigned when ZoneEquipConfig is set up.  Then any terminal unit that's connected to that AirDistUnitCool/Heat will grab and use that one.  There is a unique TermUnitSizingIndex for every zone inlet node - so some which are only used by zone equipment will never be used.